datadog/resource_datadog_integration_aws_log_collection_test.go

```diff
@@ -0,0 +1,91 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/zorkian/go-datadog-api"
+	"strings"
+	"testing"
+)
+
+const testAccDatadogIntegrationAWSLogCollectionConfig = `
+resource "datadog_integration_aws" "account" {
+  account_id                       = "1234567890"
+  role_name                        = "testacc-datadog-integration-role"
+}
+
+resource "datadog_integration_aws_log_collection" "main" {
+  account_id = "1234567890"
+  services = ["lambda"]
+  depends_on = [datadog_integration_aws.account]
+}
+`
+
+func TestAccDatadogIntegrationAWSLogCollection(t *testing.T) {
+	accProviders, cleanup := testAccProviders(t)
+	defer cleanup(t)
+	accProvider := testAccProvider(t, accProviders)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    accProviders,
+		CheckDestroy: checkIntegrationAWSLogCollectionDestroy(accProvider),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatadogIntegrationAWSLogCollectionConfig,
+				Check: resource.ComposeTestCheckFunc(
+					checkIntegrationAWSLogCollectionExists(accProvider),
+					resource.TestCheckResourceAttr(
+						"datadog_integration_aws_log_collection.main",
+						"account_id", "1234567890"),
+					resource.TestCheckResourceAttr(
+						"datadog_integration_aws_log_collection.main",
+						"services.0", "lambda"),
+				),
+			},
+		},
+	})
+}
+
+func checkIntegrationAWSLogCollectionExists(accProvider *schema.Provider) func(*terraform.State) error {
+	return func(s *terraform.State) error {
+		client := accProvider.Meta().(*datadog.Client)
+		logCollections, err := client.GetIntegrationAWSLogCollection()
+		if err != nil {
+			return err
+		}
+		for resourceType, r := range s.RootModule().Resources {
```

Comment:
Could we create helper method like in here https://github.com/terraform-providers/terraform-provider-datadog/blob/master/datadog/resource_datadog_user_test.go#L68